client/www/app/getadb/make/figma-make-instant-rules.md

```
@@ -0,0 +1,408 @@
+pnpm dlx instant-cli push perms --yes
+```
+
+# CRITICAL Query Guidelines
+
+CRITICAL: When using React make sure to follow the rules of hooks. Remember, you can't have hooks show up conditionally.
+
+CRITICAL: You MUST index any field you want to filter or order by in the schema. If you do not, you will get an error when you try to filter or order by it.
+
+Here is how ordering works:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)